material/problems/VectorGeometry/PolygonNormal/PolygonNormal.tex

```diff
@@ -1,65 +1,114 @@
-%\item $\uvecWithHat{L}$ is the unit vector from the vertex to the light source.
-%\item $\uvecWithHat{n}$ is the ``unit-normal" for the polygon (that is, the unit vector perpendicular to the polygon).
-%\end{itemize}
 %
-\\[0.5pc]
-In the figure below, the \uvecxyz{a} ``coordinate" measures of each vertex from $A_o$ are labeled.
+\\[0.45pc]
+In the figure below, the \uvecxyz{a} measures of the position vector to each
+vertex from \origin{A} are labeled.
 \\[0.0pc]
-For example, $\posvec{A_o}{P_1} \equals[\;] 3~\uvecx{a} \plus[\;] 2~\uvecy{a} \plus[\;] 0~\uvecz{a}$.
+For example, $\posvec{\origin{A}}{P_1} \equals[\;] 3~\uvecx{a} \plus[\;] 2~\uvecy{a}
+\plus[\;] 0~\uvecz{a}$.
+%
+\begin{center}
+  \includegraphicsAB[width=7cm]{polygon-solution.png}{polygon_normal.png}
+\end{center}
+%
 \begin{enumerate}
-\item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon, expressed in \uvecxyz{a}.
-\item \underline{Explain} in detail how to get \uvecWithHat{n}, the unit vector in the same direction as \bvec{n}. \underline{Sketch} \uvecWithHat{n} on the figure.
-\item \underline{Explain} why there are actually two possible \uvecWithHat{n} vectors for the polygon.
-%\item Compute \posvec{P_0}{Q}, and hence compute the unit vector $\uvecWithHat{L}$.
-%\item For a light source of strength $I_L = 0.5$, compute the diffuse illumination $I_D$.
+  \item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon,
+  expressed in terms of \uvecxyz{a}.
+    \\[0.25pc]\textbf{Result:} \\[-1.0pc]
+    %
+    \begin{displaymath}
+      \bvec{n} \equals[\;]
+      \hidemath[2cm]{4~\uvecx{a} \plus[\;] 6~\uvecy{a} \plus[\;] 8~\uvecz{a}}
+    \end{displaymath}
+    %
+    \Solution{}{0.99\linewidth}{
+      \begin{tabular}{p{0.03\linewidth}ll}
+        $\bvec{n}$
+        & $\equals[\;] \posvec{P_1}{P_2} \CrossProduct[\;] \posvec{P_1}{P_3}$
+        \\[0.45pc]
+        %\begin{align*}
+        $\posvec{P_1}{P_2}$
+        & $\equals[\;] \posvec{\origin{A}}{P_2} \minus[\;] \posvec{\origin{A}}{P_1}$
+        & $\equals[\;] (2 - 3)~\uvecx{a} \plus[\;] (0 - 2)~\uvecy{a}
+        \plus[\;] (2 - 0)~\uvecz{a}$
+        \\[0.45pc]
+        & & $\equals[\;] -1~\uvecx{a} \minus 2~\uvecy{a} \plus[\;] 2~\uvecz{a}$
+        %\end{align*}
+        \\[0.45pc]
+        $\posvec{P_1}{P_3}$
+        & $\equals[\;] \posvec{\origin{A}}{P_3} \minus[\;] \posvec{\origin{A}}{P_1}$
+        & $\equals[\;] (6 - 3)~\uvecx{a} \plus[\;] (0 - 2)~\uvecy{a}
+        \plus[\;] (0 - 0)~\uvecz{a}$
+        \\[0.45pc]
+        & & $\equals[\;] 3~\uvecx{a} \minus 2~\uvecy{a}$
+        \\[0.45pc]
+        $\bvec{n}$
+        & $\equals[\;] \posvec{P_1}{P_2} \CrossProduct[\;] \posvec{P_1}{P_3}$
+        & $\equals[\;] (-1~\uvecx{a} \minus 2~\uvecy{a} \plus[\;] 2~\uvecz{a})
+        \CrossProduct[\;] (3~\uvecx{a} \minus 2~\uvecy{a})$
+        \\[0.45pc]
+        & & $\equals[\;] -3~\uvecx{a} \DotProduct[\;] ~\uvecx{a} \plus[\;]
```

Comment:
On this line and below there are a bunch of \DotProduct that should be \CrossProduct.